### PR TITLE
Fixes #4 and #5

### LINKED ISSUE #4: Find adb on the $PATH, or allow configuring it in ~/.jpm or something
Most Android developers will have ``adb`` on their ``$PATH``.  It would be helpful to also support loading from a defaults file -- is this baked into `npm` commands in some useful way already?

### PR DESCRIPTION
**Pull request by ncalexan**

### CODE REVIEW

File: bin/jpm-mobile [hunk 1]
```diff
@@ -32,10 +33,13 @@ program
   .description("Install, run, and test the current addon in Firefox for Mobile.")
   .action(function () {
     var manifest = require(path.join(cwd, "package.json"))
-    run(manifest, makeOptions(program, "test")).then(null, console.error);
+    run(manifest, makeOptions(program, "test"))
   });
 
 function makeOptions(program, command) {
+  if (!program.adb) {
+    program.adb = require('which').sync('adb')
```
Comment (jsantell) [inline bin/jpm-mobile]: Maybe add a comment here what this does -- it's not immediately clear to me unless I were to look up `which` docs (which turns out to be the `which` unix command looks like).

Double quotes for strings here

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "dive": "0.4.0",
     "fs-extra": "0.16.4",
     "jetpack-id": "0.0.4",
-    "jpm": "1.0.0"
+    "jpm": "1.0.0",
+    "which": "~1.1.1"
```
Comment (jsantell) [inline package.json]: Bake a specific version to be consistent with the other modules (`1.1.1`)

**Comment by ncalexan** [comment]
Addressed comments, and added another thing from my local tree.

**Comment by canuckistani** [comment]
@ncalexan The travis failure appears to be bogus - should we just merge this?

**Comment by ncalexan** [comment]
@canuckistani please do, but I haven't run with it lately so I can't guarantee it's fresh.

**Comment by sanemat** [comment]
ping @jsantell @canuckistani

**Comment by kumar303** [comment]
Hi. I can merge patches. It would be great to add a test suite for this project though since I can't say for sure what might break!

**Pull request merged**